starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +18,39 @@ var words = [
   'crackpot'
 ];
 
-// Calculating a Sum
+function findLongestWord(words) {
+  var word;
+  var length = 0;
+  words.forEach(function(string){
+    if (string.length > length) {
+      length = string.length;
+      word = string;
+    };
+  });
+  return word;
+};
 
+// Calculating a Sum
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-// Calculate the Average
+function sumArray(array) {
+  return array.reduce(function (acum, current) {
+    return acum + current;
+  }, 0);
```

Comment:
Gooood Stuff :+1:, in this case, you didn't need to start the accumulator at 0. In the first run, the accumulator will be the position 0 of the array and the current, the position 1.